test/unit/com/unifina/service/ChallengeServiceSpec.groovy

```diff
@@ -1,70 +1,29 @@
 package com.unifina.service
 
-import com.unifina.domain.security.Challenge
+import com.unifina.security.Challenge
 import grails.test.mixin.Mock
 import grails.test.mixin.TestFor
+import org.joda.time.DateTime
 import spock.lang.Specification
 
 /**
  * See the API for {@link grails.test.mixin.services.ServiceUnitTestMixin} for usage instructions
  */
 @TestFor(ChallengeService)
-@Mock(Challenge)
 class ChallengeServiceSpec extends Specification {
```

Comment:
How can this work as a unit test? Are you sure it's not connecting to redis? If it is, should be an integration test